### PR TITLE
Bugfix FIXOS-8891 [Onboarding Customization] Button stackview constraints issues

### PR DESCRIPTION
**Pull request by adudenamedruby**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-8801)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/19510)

## :bulb: Description
Update a few consttraints and fix the title not appearing.

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [x] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed, I updated documentation / comments for complex code and public methods
- [x] If needed, added a backport comment (example `@Mergifyio backport release/v120`)

### CODE REVIEW

**Comment by adudenamedruby** [comment]
@Mergifyio backport release/v125

File: firefox-ios/Client/Frontend/Onboarding/Views/OnboardingMultipleChoiceButtonView.swift [hunk 1]
```diff
@@ -39,19 +39,19 @@ class OnboardingMultipleChoiceButtonView: UIView, Themeable {
...
         imageView.accessibilityIdentifier = "\(self.viewModel.a11yIDRoot)ImageView"
         imageView.isAccessibilityElement = false
     }
 
     private lazy var titleLabel: UILabel = .build { label in
-        label.numberOfLines = 0
         label.textAlignment = .center
         label.adjustsFontForContentSizeCategory = true
-        label.font = DefaultDynamicFontHelper.preferredBoldFont(withTextStyle: .body, size: 13)
+        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body, size: 13)
```
Comment (cyndichin) [inline firefox-ios/Client/Frontend/Onboarding/Views/OnboardingMultipleChoiceButtonView.swift:49]: Although `DefaultDynamicFontHelper` was used prior, while we are here, what do you think about replacing it with `FXFontStyles` instead?

File: firefox-ios/Client/Frontend/Onboarding/Views/OnboardingMultipleChoiceButtonView.swift [hunk 2]
```diff
@@ -61,6 +61,13 @@ class OnboardingMultipleChoiceButtonView: UIView, Themeable {
         imageView.isAccessibilityElement = false
     }
 
+    private lazy var stackView: UIStackView = .build { stackView in
+        stackView.axis = .vertical
+        stackView.alignment = .center
+        stackView.distribution = .equalSpacing
+        stackView.spacing = 7
```
Comment (cyndichin) [inline firefox-ios/Client/Frontend/Onboarding/Views/OnboardingMultipleChoiceButtonView.swift:68]: If my memory didn't fail me, I recalled that `7` was arbitrary when we discussed yesterday, is that still true? 

Side thought: maybe we can standardize spacing in our app... something to thing about once we standardize fonts...

File: firefox-ios/Client/Frontend/Onboarding/Views/OnboardingMultipleChoiceButtonView.swift [hunk 3]
```diff
@@ -98,26 +105,23 @@ class OnboardingMultipleChoiceButtonView: UIView, Themeable {
...
+                stackView.trailingAnchor.constraint(equalTo: containerView.trailingAnchor),
 
                 button.topAnchor.constraint(equalTo: containerView.topAnchor),
-                button.leadingAnchor.constraint(equalTo: containerView.leadingAnchor),
+                button.leadingAnchor.constraint(equalTo: imageView.leadingAnchor),
                 button.bottomAnchor.constraint(equalTo: containerView.bottomAnchor),
-                button.trailingAnchor.constraint(equalTo: containerView.trailingAnchor),
+                button.trailingAnchor.constraint(equalTo: imageView.trailingAnchor),
 
+                containerView.widthAnchor.constraint(equalToConstant: 80),
```
Comment (cyndichin) [inline firefox-ios/Client/Frontend/Onboarding/Views/OnboardingMultipleChoiceButtonView.swift:124]: nit: add magic constant to UX enum / struct?

**Review by cyndichin — APPROVED** [review]
few comments for consideration, nothing blocking! The multi-choice option looks so good!! approved based on code / live coding, Will let QA test on the devices, but let me know if you also want to pull and test around. thanks!

File: firefox-ios/Client/Frontend/Onboarding/Views/OnboardingMultipleChoiceButtonView.swift (hunk 1 above)
Comment (adudenamedruby) [reply firefox-ios/Client/Frontend/Onboarding/Views/OnboardingMultipleChoiceButtonView.swift:49]: Oh yes, totally forgot.

File: firefox-ios/Client/Frontend/Onboarding/Views/OnboardingMultipleChoiceButtonView.swift (hunk 2 above)
Comment (adudenamedruby) [reply firefox-ios/Client/Frontend/Onboarding/Views/OnboardingMultipleChoiceButtonView.swift:68]: This was arbitrary, but I checked with Gabby and she's fine with it. However, I should move this into the constants.

File: firefox-ios/Client/Frontend/Onboarding/Views/OnboardingMultipleChoiceButtonView.swift (hunk 3 above)
Comment (adudenamedruby) [reply firefox-ios/Client/Frontend/Onboarding/Views/OnboardingMultipleChoiceButtonView.swift:124]: exactly. good point

**Pull request merged**